app/code/Magento/MediaGalleryApi/Model/Asset/Command/GetByPathInterface.php

```diff
@@ -10,14 +10,14 @@
 
 /**
  * A command represents the get media gallery asset by using media gallery asset path as a filter parameter.
+ * @api
  */
 interface GetByPathInterface
 {
     /**
      * Get media asset list
      *
      * @param string $mediaFilePath
-     *
      * @return \Magento\MediaGalleryApi\Api\Data\AssetInterface
      */
     public function execute(string $mediaFilePath): \Magento\MediaGalleryApi\Api\Data\AssetInterface;
```

Comment:
Probably `$mediaAssetPath` would be a better name. Same as in the delete command above.

Reply:
changed to `$path`